tools/gitHooks/precommit.js

```diff
@@ -59,7 +82,18 @@ $ git stash pop
     await doLicenseCommit(changedFiles);
 
     // Generate API reports
-    await doApiReportsCommit();
+    await doApiReportsCommit(changedFiles);
```

Comment:
Remove `Commit` from the function names?

Reply:
Done.